recipes/console.rb

```diff
@@ -58,7 +58,12 @@
 # install posh-git
 powershell_script 'posh-git' do
   code <<-EOH
-Install-Module posh-git
+# Bootstrap provider
+Get-PackageProvider -Name NuGet -ForceBootstrap
```

Comment:
I think that only works on PowerShell 5 -- have you tried it on PowerShell 4.0 or earlier? Ideally I'd like to maintain compatibility with 2.0, so if something's not going to work on 2.0, I'd use guard expressions to just have it skipped (there's an Ohai plugin that gets the powershell version for you so u can use it in the expression easily if you want).